pkgs/servers/roon-server/default.nix

```diff
@@ -0,0 +1,60 @@
+{ alsaLib, alsaUtils, cifs-utils, fetchurl, ffmpeg, libav, mono, stdenv }:
+
+stdenv.mkDerivation rec {
+  name = "roon-server";
+  version = "100600401";
+
+  src = fetchurl {
+    url = "http://download.roonlabs.com/updates/stable/RoonServer_linuxx64_${version}.tar.bz2";
+    sha256 = "121mmdh35q4bpgsqhcps6a6q1f4ld9v4hq9gp181bf2n779pk8sh";
+  };
+
+  installPhase = ''
```

Comment:
please ```runHook preInstall``` (and postInstall at the end of the phase) when overriding the whole phase. This just makes things nicer for someone else who wants to make minor adjustments.

Reply:
Done.